### DISCUSSION TITLE
It is possible to change "symbols" naming as cli params?

### DISCUSSION THREAD

**Discussion by MadL1me** [dotnet/templating#4934] [General]
Hello. I'm curious, is there an option to set concrete name for cli param associated with an symbol? 
For example, if I have symbol "SomeTestSymbol", then in `dotnet new` it will appear as "-S|SomeTestSymbol". Can I for example change "-S" shortcut to smth else (for example lowercase it: "-s" or remove at all)? Can I change "--SomeTestSymbol" to mine defined value, for example "--someTest"?

If this feature is not implemented, I would be happy to contribute with issue and pull request

**Comment by vlada-shubina** [comment]
@MadL1me  yes, it is possible by adding `dotnetcli.host.json` file to `.template.config` folder. It allows to specify the long (`--`) and short (`-`) name to be used. 

The syntax is below:
```json
{
  "$schema": "http://json.schemastore.org/dotnetcli.host",
  "symbolInfo": {
    "SymbolNameFromTemplateJson": {
      "longName": "desired-long-option-name",
      "shortName": "on"
    },
  },
}

```

To remove short name, set it to empty string.

See example here: https://github.com/dotnet/templating/blob/main/template_feed/Microsoft.DotNet.Common.ProjectTemplates.7.0/content/ClassLibrary-CSharp/.template.config/dotnetcli.host.json

**Reply by MadL1me** [reply]
Thanks you!
